src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.test.js

```diff
@@ -8,6 +9,14 @@ import getVariantRedirectUrl from './index';
 
 const serviceNames = Object.keys(servicesWithVariants);
 
+const variantCookieConfig = {
+  ukchina: 'chinese',
+  zhongwen: 'chinese',
+};
+
```

Comment:
We already define this mapping in [`src/app/contexts/UserContext/cookies/index.js`](https://github.com/bbc/simorgh/pull/5676/files#diff-3f372212a53acbd8715a4f9d52956144R13). If you're going to re-use this, we should ensure we have a single place where this is defined and exported for use everywhere.